src/anshu/FindElementAndClickTestCase.java

```diff
@@ -0,0 +1,153 @@
+package anshu;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import com.sun.org.apache.xerces.internal.util.SynchronizedSymbolTable;
+
+public class FindElementAndClickTestCase {
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver","./resources/windows/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///C:/Users/hp/Desktop/Anshu/TechnoGitProject/SeleniumAssignment_1.html");
+		
+	     /// first name
+		
+		WebElement firstNametext = driver.findElement(By.id("first name"));
+		 firstNametext.sendKeys("Anshu");
+	     System.out.println( firstNametext.getAttribute("value"));
+		  
+		  // lastName
+		  
+		 WebElement lastNametext = driver.findElement(By.id("last name"));
+		 lastNametext.sendKeys("Kashyap");
+		  System.out.println(lastNametext.getAttribute("value"));
+		 
+		 // email
+		 
+		 WebElement emailText = driver.findElement(By.id("E-mail"));
+		 emailText.sendKeys("anshu.v.kumari02@gamil.com");
+		  System.out.println(emailText.getAttribute("value"));
+		 
+		 //companyName
+		 
+		 WebElement companyNameText = driver.findElement(By.id("Company Name"));
+		 companyNameText.sendKeys("TechnoCredits");
+		 System.out.println(companyNameText.getAttribute("value"));
+		 
+		 // gender
+		 
+		 WebElement genderRadio = driver.findElement(By.id("female"));
```

Comment:
Where is the validation of given test step? for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.